modules/phase_field/src/ics/TricrystalTripleJunctionIC.C

```diff
@@ -37,6 +38,23 @@ TricrystalTripleJunctionIC::TricrystalTripleJunctionIC(const InputParameters & p
   if (_theta1 + _theta2 >= 360.0)
     mooseError("Sum of the angles must total less than 360 degrees");
 
+  // Make sure that _junction is in the domain
+  for (unsigned int i = 0; i < LIBMESH_DIM; ++i)
+  {
+    if ((_mesh.getMinInDimension(i) > _junction(i)) || (_mesh.getMaxInDimension(i) < _junction(i)))
+      mooseError("Triple junction out of bounds");
+  }
+
+  // Default junction point is the center
+  if (!parameters.isParamSetByUser("junction"))
```

Comment:
`isParamValid` would be the correct call here (`isParamSetByUser` makes sense if a default was supplied - then you can check if it was overridden by the user)